tests/conftest.py

```diff
@@ -33,6 +33,10 @@ def testpkgs(init_git: bool = False) -> Iterator[Path]:
 
                 subprocess.run(["git", "-C", tmpdirname, "init"], check=True)
                 subprocess.run(["git", "-C", tmpdirname, "add", "--all"], check=True)
+                subprocess.run(
+                    ["git", "-C", tmpdirname, "config", "commit.gpgsign", "false"],
+                    check=True,
+                )
```

Comment:
Without this the tests cannot be run by someone that has enabled gpgsign globally